test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go

```diff
@@ -979,3 +942,16 @@ func checkHardwareCapability(ctx context.Context, workerRTNodes []corev1.Node) {
 		}
 	}
 }
+
+// cmpHostedPlaneNodes compare hosted plane nodes against workerRTNodes
```

Comment:
Nit: This function name is a bit misleading to me.  It doesn't do any real comparison, it just logs node names. Not a blocker.

Reply:
Yes, sorry, it was only for debugging , will remove that debug code.